src/test/java/seedu/address/logic/commands/SaveTodoCommandTest.java

```diff
@@ -0,0 +1,110 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CATEGORY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.testutil.TypicalEateries.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalFeeds.getTypicalFeedList;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_EATERY;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_EATERY;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.eatery.Eatery;
+import seedu.address.model.eatery.Tag;
+
+
+/**
+ * Contains integration tests and unit tests for
+ * {@code SaveTodoCommand}.
+ */
+public class SaveTodoCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), getTypicalFeedList(), new UserPrefs());
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook(), getTypicalFeedList(), new UserPrefs());
+        if (model.isMainMode()) {
+            model.toggle();
+        }
+    }
+
+    @Test
+    public void execute_validIndex_success() {
```

Comment:
do you wanna rename to be unfilteredlist like the deletecommand test naming convention?